mcp_config_files_examples/claude-git-mcp-config.example.json

```
@@ -0,0 +1,37 @@
+{
+  "//": "=== LINUX / macOS Configuration ===",
```

Biome (2.3.13)

[error] The key // was already declared.

This is where a duplicated key was declared again.

This is where a duplicated key was declared again.

This is where a duplicated key was declared again.

This is where a duplicated key was declared again.

This is where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)